pkg/operator/ceph/config/datapath.go

```diff
@@ -38,16 +40,17 @@ type DataPathMap struct {
 	// HostLogDir represents Ceph's logging directory on the host. If this is empty, logs are not
 	// persisted to the host. The log dir is always /var/log/ceph. If logs are not persisted to the
 	// host, logs are not shared between containers via empty dir or any other mechanism.
-	HostLogDir string
+	//
+	// Also represents Ceph's crash reports directory on the host
```

Comment:
Might expand this: `HostLogAndCrash dir represents Ceph's logging and crash dump dir on the host. Logs are stored in the "log" subdir and crash dumps in the "crash" subdir of this directory. If this is empty... blah blah blah.`